reagurk/public/index.html

```diff
@@ -21,6 +21,20 @@
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo.png" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <title>Gurkult</title>
+    <meta charSet="utf-8" />
+    <title>Gurkult | Open Source Community</title>
+    <meta name="description" content="Learn open source by contributing to real world projects. Be a part of an inclusive, international community of developers." />
+
+    <meta property="og:title" content="Gurkult | Open Source Community" />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://www.gurkult.com/" />
+    <meta property="og:image" content="%PUBLIC_URL%/logo.png" />
```

Comment:
What size is this image? I'm afraid that it'll render as a huge "banner" instead of a thumbnail. That would be bad. I hate huge banners for things like logos.

Reply:
@gustavwilliam can you tell me what size should be there?how to fix

Reply:
@gustavwilliam The image was same , i just moved the code from what it was earlier written , also the other 2 reviewers were fine with this changes , Infact it was checked by 2 of them , Let me know if you want to change anything

Reply:
> 



> What size is this image? I'm afraid that it'll render as a huge "banner" instead of a thumbnail. That would be bad. I hate huge banners for things like logos.

Afaik, the `summary_small_image` should fix that.

Reply:
@dawnofmidnight The pull request is approved by arnav , can you please review and approve so that it can get merge.

Reply:
> @gustavwilliam The image was same , i just moved the code from what it was earlier written , also the other 2 reviewers were fine with this changes , Infact it was checked by 2 of them , Let me know if you want to change anything

When I review, I might find something that doesn't look good to me, even though it's fine to other reviewers. That's why we have multiple reviewers.

The image might not strictly be a part of your PR, but I still think it's important to address.